### ISSUE TITLE
Creating Stack fails in AWS

### ISSUE THREAD

**Issue by kesrw38** [asishrs/smartthings-ringalarmv2#41]
Hello,

Has anyone else had any trouble creating the ringalarm-gateway.yaml stack. I keep getting the below error message.

Error occurred while GetObject. S3 Error Code: NoSuchBucket. S3 Error Message: The specified bucket does not exist (Service: AWSLambdaInternal; Status Code: 400; Error Code: InvalidParameterValueException; Request ID: d19f49eb-9da6-11e9-bfe6-2bd79cd1146a)

**Comment by Its2loud1** [comment]
I think there is something wrong with the AWS integration.

**Comment by asishrs** [comment]
I will check this in weekend.

**Comment by Its2loud1** [comment]
Thanks! Appreciate it.

**Comment by asishrs** [comment]
I did create a test S3 bucket and tried deployment. It is successful. It seems Amazon changed the interface for S3 bucket as well as CloudFormation template but if you make only changes according to the instruction, you should be good.

**Comment by dmoses1969** [comment]
> I did create a test S3 bucket and tried deployment. It is successful. It seems Amazon changed the interface for S3 bucket as well as CloudFormation template but if you make only changes according to the instruction, you should be good.

I deleted everything on Amazon and attempted to follow the steps to set this up but it appears as though things on Amazon have changed. Looks different. Do you have updated instructions by chance?
Thanks

**Comment by dmoses1969** [comment]
> > I did create a test S3 bucket and tried deployment. It is successful. It seems Amazon changed the interface for S3 bucket as well as CloudFormation template but if you make only changes according to the instruction, you should be good.
> 
> I deleted everything on Amazon and attempted to follow the steps to set this up but it appears as though things on Amazon have changed. Looks different. Do you have updated instructions by chance?
> Thanks

As soon as I get to this step
" Wait for 2-5 minutes for the creation of the stack. You will see a status CREATE_COMPLETE once your stack is successfully created. "
It shows, created and immediately shows deleted.

**Comment by asishrs** [comment]
I can update the instructions in sometime.

**Comment by dmoses1969** [comment]
> I can update the instructions in sometime.

Here's the problem I am having
![image](https://user-images.githubusercontent.com/10082767/78542717-30d4d400-77bd-11ea-9fde-d4726cade32c.png)

**Comment by asishrs** [comment]
@dmoses1969 Check this. I will add this to the instructions.

![cloud-formation](https://user-images.githubusercontent.com/12078528/78619395-2b65a100-7832-11ea-89ba-1ca58e0a7b7f.gif)

**Comment by Its2loud1** [comment]
Asish, thank you for fixing this integration during this difficult time. I hope continue to stay healthy. 

For anyone having issues with 2FA sticking in the Smart App. I recommend deleting the Ring Manager from ST Classic App and re-adding the app.  Once I did this, I entered my info back into Ring Manager, turned on 2FA, entered the texted code. Selected view Ring Status. Made sure everything populated. Tapped Done and Save out of Ring Manager. After a few minutes, everything is working normal. 

I don’t think the Ring Manager likes it if you get out of the Classic app to check the texted code. Use another device to setup Ring Manager while you get texted the code on your phone.
